itest/lnd_multi-hop_force_close_test.go

```diff
@@ -530,6 +533,25 @@ func runLocalClaimOutgoingHTLC(ht *lntest.HarnessTest,
 	// Once this transaction has been confirmed, Bob should detect that he
 	// no longer has any pending channels.
 	ht.AssertNumPendingForceClose(bob, 0)
+
+	// Now that Bob has claimed his HTLCs, Alice should mark the two
+	// payments as failed.
+	//
+	// Alice will mark this payment as failed with no route as the only
+	// route she has is Alice->Bob->Carol. This won't be the case if she
+	// has a second route, as another attempt will be tried.
+	//
+	// TODO(yy): we should instead mark this payment as timed out if she has
+	// a second route to try this payment, which is the timeout set by Alice
+	// when sending the payment.
```

Comment:
Nit: So you mean we should add more routes to this test-case ?

Reply:
yeah we should test the behavior that Alice's node tried to send this payment again but failed.